config/database.yml~

```diff
@@ -0,0 +1,71 @@
+## edit these for local development
```

Comment:
Tento subor (`database.yml~`), obrazky (`pics.zip`, adresar `pics`), data otazok (`*.csv`) nemaju byt commitnute a pushnute - treba ich vymazat.
Ked ich mas lokalne v adresari, tak ich nepridavaj do gitu (pripadne by sa to mohlo zaroven dat do `.gitignore`